examples/common/builders.py

```diff
@@ -15,6 +16,10 @@
 from composer.optim.scheduler import (ConstantWithWarmupScheduler,
                                       CosineAnnealingWithWarmupScheduler,
                                       LinearWithWarmupScheduler)
+from omegaconf import DictConfig
```

Comment:
can combine these imports on to one line

Reply:
linter wont allow it...

Reply:
lol fine. linter wins